plugins/woocommerce/src/StoreApi/Utilities/OrderController.php

```diff
@@ -194,10 +194,11 @@ protected function perform_custom_order_validation( \WC_Order $order ) {
 		 *
 		 * @param \WC_Order $order             The order object.
 		 * @param \WP_Error $validation_errors WP_Error object to add custom errors to.
+		 * @since 9.3.4
```

Comment:
```suggestion
		 * @since 9.5.0
```

This will likely land in 9.5.0 given the code freeze for 9.4.0 passed already.